### DISCUSSION TITLE
[Feature Request] Light Mode & Device Default Theme

### DISCUSSION THREAD

**Discussion by seanrobleto** [ChristopherKlay/StadiaEnhanced#120] [Ideas & Requests]
Hello Christopher, hope you are doing well. Although not super important, I think it would be really cool to have an option for a light mode for the Stadia UI along with the options for dark mode and device default theme if possible. Thank you for your work!

**Comment by ChristopherKlay** [comment]
Will add it to the project board.

I'm not sure how doable this is due to the high amount of changes on Stadias end most likely breaking it every couple of weeks, or even days, but i'll look into it (:
